### PR TITLE
#7 Create Entity Model Classes

### PR DESCRIPTION
**Pull request by noor848**

### CODE REVIEW

File: Conduit/Conduit/bin/Debug/net6.0/appsettings.Development.json
Comment (hayatzsala) [inline Conduit/Conduit/bin/Debug/net6.0/appsettings.Development.json:1]: @noor848 again, the bin folder should be in .gitignore

File: Conduit/Conduit/bin/Debug/net6.0/appsettings.json
Comment (hayatzsala) [inline Conduit/Conduit/bin/Debug/net6.0/appsettings.json:1]: .gitignore

File: Conduit/Conduit.Db/Article.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Article.cs]: @noor848 why not Guid or long? not recommended to use int as id

File: Conduit/Conduit.Db/Comment.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Comment.cs]: @noor848 same as articleId

File: Conduit/Conduit.Db/Article.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Article.cs]: @noor848 actually there is no need for these collection, we replace it by using separate Table/mode

File: Conduit/Conduit.Db/ConduitContext.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/ConduitContext.cs:17]: @noor848 do you create the DB from SQL files? if yes , can you attached them in folder please?
if No, you can here declare the key for each table (primary/foregin)

File: Conduit/Conduit.Db/Favourite.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Favourite.cs]: @noor848 all ids here same as ArticleId comment

File: Conduit/Conduit.Db/Follower.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Follower.cs]: no need for this attribute, actually when you create new object just assign new Guid() to the DB model

File: Conduit/Conduit.Db/User.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/User.cs]: @noor848 hmmm, thinking deeply here do we have any relation with comment? NO, so no need for this, if you need to add relation so you have to add column to comment table "CreatedBy" and be a foreign key to user table

File: Conduit/Conduit.Db/User.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/User.cs]: @noor848 as a comment on Article

File: Conduit/Conduit.Db/User.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/User.cs]: same as the comment of "comment createdBy"

File: Conduit/Conduit/Program.cs
Comment (hayatzsala) [inline Conduit/Conduit/Program.cs]: hmmmm, why do I see these changes here? if you checkout the branch from the main so I shouldn't see the changes here,and you can isolate it with DB changes.

File: Conduit/Conduit.Db/Article.cs
Comment (noor848) [reply Conduit/Conduit.Db/Article.cs]: i did it for user and follower ill  change it for the rest

File: Conduit/Conduit/bin/Debug/net6.0/appsettings.Development.json
Comment (noor848) [reply Conduit/Conduit/bin/Debug/net6.0/appsettings.Development.json:1]: don in the second branch

**Comment by noor848** [comment]
Conduit #7

File: Conduit/Conduit.Db/Favourite.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/Favourite.cs]: AtricleId is Guid not int, don't forget to update it here

File: Conduit/Conduit.Db/User.cs
Comment (hayatzsala) [inline Conduit/Conduit.Db/User.cs]: clean code : remove unuseful comments

**Review by hayatzsala — APPROVED** [review]
Approved, 2 tiny comments, if you can update it before merge please

**Review by hayatzsala — APPROVED** [review]

**Review by hayatzsala — APPROVED** [review]

**Pull request merged**
